satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -84,6 +86,7 @@ datasets:
     key: '#19#brightnessTemperature'
     resolution: 48006.450653072
     wavelength: [ 3.48, 3.92, 4.36 ]
+    long_name: TOA Brightness Temperature at 3.9um (all pixels)
```

Comment:
For ASR I suggest to specify in the long name that this is a segment mean value and thus different to the L15 brightness temperature --> "TOA Brightness Temperature at 3.9um (segment mean of all pixels". 

This comment applies to all ASR entries.

Reply:
done